File: module_main/src/main/java/com/phone/module_main/main/presenter/IMainPresenter.java
package com.phone.module_main.main.presenter;

import com.trello.rxlifecycle3.components.support.RxAppCompatActivity;

import java.util.Map;

/**
 * author    : Urasaki
 * e-mail    : 1164688204@qq.com
 * date      : 2019/3/10 16:58
 * introduce :
 */

public interface IMainPresenter {

    void mainData(RxAppCompatActivity rxAppCompatActivity, Map<String, String> bodyParams);
}

File: module_main/src/main/java/com/phone/module_main/presenter/IMainPresenter.kt
package com.phone.module_main.presenter

import com.trello.rxlifecycle3.components.support.RxAppCompatActivity

/**
 * author    : Urasaki
 * e-mail    : 1164688204@qq.com
 * date      : 2020/3/10 17:15
 * introduce : Presenter登录模块的（一个小模块对应一个Presenter）
 */

interface IMainPresenter {

    fun mainData(rxAppCompatActivity: RxAppCompatActivity, bodyParams: Map<String, String>)
}